src/views/blocks/tickets/submit-button-modal.php

```diff
@@ -21,20 +24,34 @@
 $button_text = _x( 'Get Tickets', 'Get selected tickets.', 'event-tickets' );
 
 /**
- * Filter Modal Content
+ * Allow filtering of the button classes for the tickets block.
+ *
+ * @since TBD
+ *
+ * @param array $button_name The button classes.
+ */
+$button_classes = apply_filters( 'tribe_tickets_ticket_block_submit_classes', [
+	'tribe-common-c-btn',
+	'tribe-common-c-btn--small',
+	'tribe-tickets__buy'
+] );
```

Comment:
:no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: 3fb479ee694a0e05eb10b4c4cac4e67f</sub>